apps/sim-core/README.md

```diff
@@ -12,4 +12,77 @@
 
 It uses a legacy version of hEngine which is no longer maintained, separate from the primary [HASH Engine] (**hEngine**) found in this repository.
 
-_The hCore source code will be made available here shortly._
+
+## Project Status
+
+`sim-core` is currently undergoing a migration process away from our internal infrastructure and toward this open-source, self-hosted version.  While we're making it available at this time so that our users can preserve and run their preesting simulations, additional migraiton work is ongoing.  Those upcoming tasks include:
```

Comment:
```suggestion
**hCore** is currently in the process of transitioning from being closed-source and hosted on our internal infrastructure towards being a free, open-source IDE available to self-host. While we're making it available at this time so that users can continue to work with and run existing simulations, additional migration work is ongoing, and we'll be changing the way simulations are created in the future. Upcoming tasks in 'phase one' of the migration include:
```